regulation/tree.py

```diff
@@ -897,25 +897,26 @@ def build_interp_layer(root):
     """
 
     layer_dict = OrderedDict()
-    interpretations = root.find('.//{eregs}interpretations')
+    interpretations = root.findall('.//{eregs}interpretations')
 
     if interpretations is not None:
-        first_label = interpretations.get('label')
-        first_key = first_label.split('-')[0]
-        layer_dict[first_key] = [{'reference': first_label}]
-
-        interp_sections = interpretations.findall(
-            './/{eregs}interpSection')
-        interp_paragraphs = interpretations.findall(
-            './/{eregs}interpParagraph')
-        targetted_interps = [i for i in
-            interp_sections + interp_paragraphs
-            if i.get('target') is not None]
-
-        for interp in targetted_interps:
-            target = interp.get('target')
-            label = interp.get('label')
-            layer_dict[target] = [{'reference': label}]
+        for interpretation in interpretations:
+            first_label = interpretation.get('label')
+            first_key = first_label.split('-')[0]
+            layer_dict[first_key] = [{'reference': first_label}]
```

Comment:
Without truly understanding this code, what is this used for? Take for example [this Reg E RegML](https://raw.githubusercontent.com/cfpb/regulations-xml/master/regulation/1005/2016-24506.xml). It has multiple `<interpretations>` sections with labels `1005-Subpart-A-Interp`, `1005-Subpart-B-Interp`, and `1005-Appendices-Interp`.

This code would split these labels and, for each one, set `first_key = 1005`, then set `layer_dict[1005]` to something different (for example, `[{'reference': '1005-Subpart-A-Interp'}]` or `[{'reference': '1005-Subpart-B-Interp'}]`. Isn't that bad? It's re-setting the same key with different things. What is this used for?

Reply:
Welp, this actually breaks the table of contents. Gonna go back to my cave and weep.